_scicomputing/reference_training.md

```diff
@@ -14,7 +14,7 @@ There are a variety of resources for training on various aspects of bioinformati
 
 ### Community Groups
 
-[The Coop](http://thecoop.fredhutch.org/) is the Fred Hutch Bioinformatics and Data Science Cooperative, and works to share information and resources about computational work across the Hutch. The Coop maintains a listserv, calendars of data science events, and [The Coop Communities Slack](https://fhbig.slack.com/), supported by FHBig. [FHBig](https://fredhutch.github.io/FHBig/) is the Fred Hutch Bioinformatics Interest Group, a community-based group that hosts a [blog](https://fredhutch.github.io/FHBig/year-archive/) and facilitates information sharing among the bioinformatics research community at the Hutch. Hutch employees can learn more through the links above or by emailing `coophelp` wiht questions or to be added to our email/newsletter list.
+[The Coop](http://thecoop.fredhutch.org/) is the Fred Hutch Bioinformatics and Data Science Cooperative, and works to share information and resources about computational work across the Hutch. The Coop maintains a listserv, calendars of data science events, and [The Coop Communities Slack](https://fhbig.slack.com/), supported by FHBig. [FHBig](https://fredhutch.github.io/FHBig/) is the Fred Hutch Bioinformatics Interest Group, a community-based group that hosts a [blog](https://fredhutch.github.io/FHBig/year-archive/) and facilitates information sharing among the bioinformatics research community at the Hutch. Hutch employees can learn more through the links above or by emailing `coophelp` with questions or to be added to our email/newsletter list.
```

Comment:
Maybe put the actual email address instead of `coophelp`

Reply:
Our contribution/style guidelines specify this format for email addresses.

Reply:
My bad!